tools/clean/atlas/clusters.go

```diff
@@ -25,6 +25,8 @@ func (c *Cleaner) deleteClusters(ctx context.Context, projectID string, clusters
 	for _, cluster := range clusters {
 		if cluster.GetStateName() == "DELETING" {
 			fmt.Println(text.FgHiBlue.Sprintf("\t\t\tCluster %s is being deleted...", cluster.GetName()))
+
+			return
```

Comment:
@helderjs  mb you meant "continue"? I mean, shouldn't it check the rest of the clusters?

Reply:
good catch